client/go/src/cmd/query.go

```diff
@@ -29,28 +29,36 @@ var queryCmd = &cobra.Command{
         return nil
     },
     Run: func(cmd *cobra.Command, args []string) {
-        query(args[0])
+        query(args)
     },
 }
 
-func query(argument string) {
-    if ! startsByParameter(argument) { // Default parameter
-        argument = "yql=" + argument
-    }
+func query(arguments []string) {
+    var argBuilder strings.Builder;
+    for i := 0; i < len(arguments); i++ {
+        argument := arguments[i]
 
-    argument = escapePayload(argument)
-    if argument == "" {
-        return
+        if ! startsByParameter(argument) { // Default parameter
+            argument = "yql=" + argument
+        }
+
+        argument = escapePayload(argument)
+        if argument == "" {
+            return
+        }
+        if i > 0 {
+            argBuilder.WriteString("&")
+        }
+        argBuilder.WriteString(argument)
     }
 
-    path := "/search/?" + argument
+    path := "/search/?" + argBuilder.String()
```

Comment:
Consider constructing a `net.URL` from the arguments instead, so that the end result becomes a properly encoded URL.

Reply:
Better - thanks